server/questions.js

```diff
@@ -126,15 +126,21 @@ var lookupQuestionById = function(questionId, callback) {
  * Update the question object in the database and call the callback function
  * with the result of the comparison and the new question object.
  */
-exports.checkAnswer = function(questionId, userId, answer, callback) {
+exports.checkAnswer = function(questionId, user, answer, callback) {
```

Comment:
indentation in this function seems kinda off, just take another look at it :)

Reply:
fixed